### ISSUE TITLE
Can you provide the specific track indices for ATAC, H3K27ac, H3K4me1, and TF binding sites in bigwig_tracks_logits output

### ISSUE THREAD

**Issue by yuyadanyadan** [instadeepai/nucleotide-transformer#147]
The bigwig_tracks_logits output of the post-trained model is a tensor of shape [batch, length, num_tracks], and the specific track indices for ATAC, H3K27ac, H3K4me1, and TF binding sites cannot be directly found in the NTv3 model outputs.
Thank you so much

**Comment by bernardo-de-almeida** [comment]
Hi, thanks for the question. The index → assay mapping is a two-step lookup:

**1. Index → track name.** The last axis of `bigwig_tracks_logits` follows the order of `config.bigwigs_per_species[species]`:

```python
names = config.bigwigs_per_species["human"]   # ordered list, len == last-axis size for human
track_idx = names.index("ENCSR000DWD")          # -> 1536
track = bigwig_tracks_logits[:, :, track_idx]
```

(See `inference_posttrained.ipynb` in the repo.) The mapping is per-species (the species token selects the head), and the last axis is zero-padded to the largest species (human, 7362), so for other species only the first `len(names)` indices are valid.

**2. Track name → assay/target.** The names are experiment accessions and don't encode the assay. That metadata lives in `data/functional_tracks_metadata.csv` in the [`InstaDeepAI/ntv3_tracks` Space](https://huggingface.co/spaces/InstaDeepAI/ntv3_tracks/blob/main/data/functional_tracks_metadata.csv). You can read it straight from the URL, no download needed:

```python
import pandas as pd

meta = pd.read_csv(
    "https://huggingface.co/spaces/InstaDeepAI/ntv3_tracks/resolve/main/data/functional_tracks_metadata.csv"
)
# columns: file_id, biosample_type, tissue, assay, strand, experiment_target, specie, dataset

names = config.bigwigs_per_species["human"]
sp = meta[meta.specie == "human"]

atac    = sp[sp.assay == "ATAC-seq"].file_id                 # 232 tracks
h3k27ac = sp[sp.experiment_target == "H3K27ac"].file_id      # 225 tracks
h3k4me1 = sp[sp.experiment_target == "H3K4me1"].file_id      # 255 tracks
tf      = sp[sp.assay == "TF ChIP-seq"].file_id              # 1833 tracks (target = TF name)

nameset = set(names)
atac_idx = {acc: names.index(acc) for acc in atac if acc in nameset}
```

For example the H3K27ac track `ENCSR391NPE` (22Rv1) is at index `names.index("ENCSR391NPE")` -> 222.


Let me know if this answers your question, thanks.
